udaykalra/portfolio/target/portfolio-1/style.css

```diff
@@ -1,9 +1,144 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.bottomRow
+{
+	padding-top: 30px;
 }
```

Comment:
Style recommends placing the opening brace on the same line as the selector:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Block_Separation

in this case:
```
.bottomRow {
  padding-top: 30px;
}
```

Reply:
This has been implemented in latest push.